README.md

```diff
@@ -96,15 +103,41 @@ simple
 
     *Example where we have invalid ICU message syntax.*
     
-    **(ascii cinema link)**
-    <a align="center" href="https://asciinema.org/a/jIu2dJVuVucawqaFmgjuSUPg0" target="_blank"><img src="https://asciinema.org/a/jIu2dJVuVucawqaFmgjuSUPg0.png" /></a>
+    ![](assets/invalid-icu-syntax-screenshot.png)
 
 ## Examples
 Check out the [Examples](examples/) folder to see different use cases.
 
 ## Configuring the rules
 - Simply update your `.eslintrc.*` with overrides for the individual rules.
 - Eslint severities: 2 = error, 1 = warning, 0 = off
+- Example of the module's default rule configuration:
```

Comment:
Are the example-related changes part of the formatter work?  If not, consider pulling them into a separate PR.

Reply:
yes, for the most part they are. (although, I did extend an example)